grl/policy_eval.py

```diff
@@ -110,10 +115,9 @@ def create_td_model(self, occupancy):
 
                 # R
                 with np.errstate(invalid='ignore'):
-                    R_contributions = np.nan_to_num(self.amdp.R * T_contributions / T_obs_obs[:,curr_ob,next_ob])
+                    R_contributions = np.nan_to_num(self.amdp.R * T_contributions / T_obs_obs[:,curr_ob,next_ob][:, None, None])
```

Comment:
Is this to make the denominator the same shape as `T_contributions`?

Reply:
Same dimensions yea